### PR TITLE
Add recipes for dramatiq [skip appveyor]

### PR DESCRIPTION
**Pull request by lincrosenbach**

### CODE REVIEW

File: recipes/prometheus-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+about:
+  home: https://github.com/prometheus/client_python
+  license: Apache Software
+  license_family: APACHE
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE' 
+  summary: Python client for the Prometheus monitoring system.
+
+extra:
+  recipe-maintainers:
+    - nicoddemus
```
Comment (nicoddemus) [inline recipes/prometheus-client/meta.yaml]: 👍

**Comment by marcelotrevisani** [comment]
Tests failed for osx
Maybe it has some dependencies regarding OS, I will take a look on that later. If it's true this recipe can't be noarch
@igortg , please do not merge that yet

**Comment by jjhelmus** [comment]
[prometheus_client](https://github.com/conda-forge/prometheus_client-feedstock) already exists in `conda-forge` and `defaults`.  I do not think we should have two packages with slightly different names.

**Comment by nicoddemus** [comment]
> prometheus_client already exists in conda-forge and defaults.

Oh, we looked for `prometheus-client-feedstock`, which doesn't exist, so we concluded it wasn't available in conda-forge yet.

We will fix the recipe ASAP.

**Comment by lincrosenbach** [comment]
Removed prometheus-client recipe.

**Comment by lincrosenbach** [comment]
Not really clear to me what this MacOSX build failure means:

```
dyld: lazy symbol binding failed: Symbol not found: _getentropy
  Referenced from: /Users/travis/miniconda/conda-bld/dramatiq_1538408896333/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/bin/python
  Expected in: /usr/lib/libSystem.B.dylib
dyld: Symbol not found: _getentropy
  Referenced from: /Users/travis/miniconda/conda-bld/dramatiq_1538408896333/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/bin/python
  Expected in: /usr/lib/libSystem.B.dylib
```

Any hints? Please note that `dramatiq` is a pure-python library.

**Comment by lincrosenbach** [comment]
The issue does not seems related to the package itself. It is related to a known issue with XCode as can be seen in the links below: 

* https://github.com/yousseb/meld/issues/62
* https://github.com/Homebrew/homebrew-core/issues/4809#issuecomment-247021138

**Comment by marcelotrevisani** [comment]
The errors regarding abi are know, but I thought that was fixed.

It is related to
https://github.com/conda-forge/python-feedstock/issues/201

**Comment by nicoddemus** [comment]
@marcelotrevisani Circle has passed, so I think we can merge this now. 👍

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks for adding this in ``conda-forge``! :tada:

**Pull request merged**